README.md

```diff
@@ -1 +1,135 @@
-# jx-app-jacoco
\ No newline at end of file
+# jx-app-jacoco
+
+jx-app-jacoco provides a means for transferring a Jacoco XML code coverage report 
+from a [Jenkins X](https://jenkins-x.github.io/jenkins-x-website/) build to 
+a fact in the PipelineActivity CRD.
+
+You must have a Jenkins X cluster to install and use the jx-app-jacoco app.  If
+you do not have a Jenkins X cluster and you would like to try it out, the 
+[Jenkins X Google Cloud Tutorials](https://jenkins-x.io/getting-started/tutorials/) 
+is a great place to start.
+
+## Installation
+
+Using the [jx command line tool](https://jenkins-x.io/getting-started/install/), run 
+the following command.
+
+`jx add app -n "" jx-app-jacoco --repository "http://chartmuseum.jenkins-x.io"`
+
+NOTE: The syntax of this command is evolving and will change.
+
+Upon successful installation, you should see jx-app-jacoco in the list of pods 
+running in your cluster.
+
+`kubectl get pods`
+
+## Usage
+
+The current usage of jx-app-jacoco is limited to Maven projects.  You must 
+configure the build section of your Maven POM file for Jacoco to generate
+an XML report in addition to the, default, jacoco.exec file.
+
+Example Maven POM file:
+
+    <build>
+      <plugins>
+         <plugin>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-maven-plugin</artifactId>
+         </plugin>
+         <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.8.2</version>
+            <executions>
+               <execution>
+                  <id>default-prepare-agent</id>
+                  <goals>
+                     <goal>prepare-agent</goal>
+                  </goals>
+               </execution>
+               <execution>
+                  <id>prepare-xml-report</id>
+                  <goals>
+                     <goal>report</goal>
+                  </goals>
+                  <phase>verify</phase>
+               </execution>
+            </executions>
+         </plugin>
+      <plugin>
+     </plugins>
+    </build>
+
+Ensure that your Jenkinsfile includes the following command so the Jacoco XML report is 
+collected.
+
+`sh "jx step collect --pattern=target/site/jacoco/jacoco.xml --classifier=jacoco"`
```

Comment:
```bash or ```console to get the markdown and formatting right

Reply:
I think it is nice to settle on one way of code block formatting.

Reply:
I will be consistent with the jx repo since it is likely based on internal doc standards.  Their approach is indentation (a tab).